test/jdk/ProblemList.txt

```diff
@@ -258,6 +258,7 @@ java/awt/Component/GetScreenLocTest/GetScreenLocTest.java 4753654 generic-all
 java/awt/Component/SetEnabledPerformance/SetEnabledPerformance.java 8165863 macosx-all
 java/awt/Clipboard/HTMLTransferTest/HTMLTransferTest.java 8017454 macosx-all
 java/awt/Frame/MiscUndecorated/RepaintTest.java 8266244 macosx-aarch64
+java/awt/Menu/MenuVisibilityTest.java 8161110 macosx-all
```

Comment:
@prrace  Should it be moved to `Client manual tests` section ?

Reply:
I think yes, I've updated. Can you please re-review and approve if its fine.